frontend/src/components/UserList.js

```diff
@@ -0,0 +1,174 @@
+    if (sorted) {
+      if (direction) {
+        return <img src="/img/up_arrow.svg" alt="Up Arrow" />;
+      }
+
+      return <img src="/img/down_arrow.svg" alt="Down Arrow" />;
+    }
+
+    return <img src="/img/updown_arrow.svg" alt="upDown Arrow" />;
+  };
+
+  /**
+   * Returns the header element to be displayed depending on the position in the table
+   * @param {*} column A column acquired with react table methods
+   * @param {int} index The current location in the table header
+   * @returns
+   */
+  const getHeader = (column, index) => {
+    // Start of the header
+    if (index === 0) {
+      return (
+        <th
+          className="people_table_header_start"
+          {...column.getHeaderProps(column.getSortByToggleProps())}
+        >
+          {column.render("Header")}
+          <span className="sort_toggle">{getArrowImage(column.isSorted, column.isSortedDesc)}</span>
+        </th>
+      );
+    }
+
+    // Uncomment lines (82 and 85) and (94 and 97) to get sorting to work for the middle and end headers
+    // End of the header
+    if (index === columns.length - 1) {
+      return (
+        <th
+          className="people_table_header_end"
+          // {...column.getHeaderProps(column.getSortByToggleProps())}
+        >
+          {column.render("Header")}
+          {/* <span className="sort_toggle">{getArrowImage(column.isSorted, column.isSortedDesc)}</span> */}
+        </th>
+      );
+    }
+    // Header elements in the middle
+
+    return (
+      <th
+        className="people_table_header"
+        // {...column.getHeaderProps(column.getSortByToggleProps())}
+      >
+        {column.render("Header")}
+        {/* <span className="sort_toggle">{getArrowImage(column.isSorted, column.isSortedDesc)}</span> */}
+      </th>
+    );
+  };
+
+  const getCell = (cell, colIndex, rowIndex) => {
+    if (colIndex === 0) {
+      return (
+        <td
+          {...cell.getCellProps()}
+          style={
+            rowIndex % 2 === 0
+              ? { background: "white" }
+              : { background: "rgba(187, 188, 188, 0.2)" }
+          }
+          className="people_table_data_start"
+          key={Math.random()}
+        >
+          {cell.render("Cell")}
+        </td>
+      );
+    }
+
+    if (colIndex === columns.length - 1) {
+      return (
+        <td
+          {...cell.getCellProps()}
+          style={
+            rowIndex % 2 === 0
+              ? { background: "white" }
+              : { background: "rgba(187, 188, 188, 0.2)" }
+          }
+          className="people_table_data_end"
+          key={Math.random()}
+        >
+          {cell.render("Cell")}
+        </td>
+      );
+    }
+
+    return (
+      <td
+        {...cell.getCellProps()}
+        style={
+          rowIndex % 2 === 0 ? { background: "white" } : { background: "rgba(187, 188, 188, 0.2)" }
+        }
+        className="people_table_data"
+        key={Math.random()}
+      >
+        {cell.render("Cell")}
```

Comment:
React's also complaining about keys right here, but I have no idea how to fix it.

[This is a thread about this exact issue](https://spectrum.chat/react-table/general/rendering-custom-cells-the-correct-way~ea55cb63-4a7d-4d16-be94-24942e99fffd), but it's not really the end of the world if we can't fix it.